pyproject.toml

```diff
@@ -250,6 +250,7 @@ Pympler = { version = "*", optional = true }
 parameterized = { version = ">=0.7.4", optional = true }
 idna = { version = ">=2.5", optional = true }
 pyicu = { version = ">=2.10.2", optional = true }
+requests-unixsocket = "^0.3.0"
```

Comment:
I'm not really sure how i feel about using this package -- it is considered "alpha" software according to the metadata. I'm wondering if it is better to just limit the package to using HTTP. The `register_new_matrix_user` script can be used externally via HTTP already.

I.e. I'm wondering if we should just document it as a limitation instead?

Reply:
Well, this is up for you guys to decide. I just thought to suggest this change, as it seemed to be a straight-forward thing to do with this package.